BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/ble/beacon/signal/RssiFilter.java

```diff
@@ -16,4 +17,8 @@
 
     void setMaximumTimestamp(long maximumTimestamp);
 
+    long getDuration();
```

Comment:
These values are only available in classes that extend `WindowFilter` and thus shouldn't be part of this interface.